packages/datatrak-web/src/components/Icons/CommentIcon.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import { SvgIcon, SvgIconProps } from '@material-ui/core';
+
+export const CommentIcon = (props: SvgIconProps) => {
```

Comment:
Thanks for updating this. I was not sure if the MUI one was close enough and I think this one does look nicer